include/wx/gtk/dc.h

```diff
@@ -42,6 +42,9 @@ class wxGTKCairoDCImpl: public wxGCDCImpl
     // Set m_size from the given (valid) GdkWindow.
     void InitSize(GdkWindow* window);
 
+    // Adjust cairo context for RTL.
+    void AdjustCairoContext(cairo_t* cr);
```

Comment:
If it's only going to be used for RTL, I'd prefer to use a more explicit name for it, e.g. `ApplyLayoutDirection()`.